packages/json-api-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@disruptph/json-api-utils",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
No need na mag update sa version manually since ang `yarn run version` command na ang bahala sa pag update sa versions sa tanan affected packages. Though dapat ra ni i-run ang command sa master branch once ma merge na ang PR